### ISSUE TITLE
`prefer-sveltekit-response-helpers`

### ISSUE THREAD

**Issue by benmccann** [sveltejs/eslint-plugin-svelte#1217]
### Motivation

See https://github.com/lucia-auth/example-sveltekit-google-oauth/pull/1 for an example

### Description

Prefer using `error` and `redirect` over using `Response`. There might be some advanced cases where `Response` is required, but it shouldn't be for simple ones

### Examples

```ts
<!-- ✓ GOOD -->
```
redirect(307, "/");
```

<!-- ✗ BAD -->
return new Response(null, {
	status: 302,
	headers: {
		Location: "/"
	}
});
```
